html/forms/tasks/basic-controls/basic-controls1.html

```diff
@@ -30,8 +30,6 @@
 
     </section>
 
-    <textarea class="playable playable-css" style="height: 130px;">
-
     </textarea>
```

Comment:
@wbamberg Arrrgggg. Missed the end tags. Also, might be better to make as HTML comment because that way it is clear that you explicitly disabled the box rather than omitted it

Reply:
Fixed in https://github.com/mdn/learning-area/pull/589